Header header
string object_class
float32 confidence
int32 x
int32 y
int32 width
int32 height
string material